app/src/main/java/org/oppia/android/app/databinding/ConstraintViewAdapters.java

```diff
@@ -0,0 +1,42 @@
+package org.oppia.android.app.databinding;
+
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.constraintlayout.widget.ConstraintLayout;
+import androidx.constraintlayout.widget.ConstraintSet;
+import androidx.databinding.BindingAdapter;
+
+/** Holds all custom binding adapters that set constraints to views. */
```

Comment:
Just ``Custom binding adapters that set constraints to views.`` is sufficient.

Reply:
Done